rules-tests/TypeDeclaration/Rector/FunctionLike/ReturnTypeDeclarationRector/Fixture/preserve-generator.php.inc

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\FunctionLike\ReturnTypeDeclarationRector\Fixture;
+
+final class DemoFile
+{
+    /**
+     * @return \Generator<int>
+     **/
+    public function generator()
+    {
+        yield 5;
+    }
+}
+?>
+-----
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\FunctionLike\ReturnTypeDeclarationRector\Fixture;
+
+final class DemoFile
+{
+    /**
+     * @return \Generator<int>
+     **/
+    public function generator(): \Generator
+    {
+        yield 5;
+    }
+}
+?>
```

Comment:
There should be one more fixture to add return type without any docblock.

Reply:
There are already fixture for that:

- https://github.com/rectorphp/rector-src/blob/main/rules-tests/TypeDeclaration/Rector/FunctionLike/ReturnTypeDeclarationRector/Fixture/generator.php.inc

- https://github.com/rectorphp/rector-src/blob/main/rules-tests/TypeDeclaration/Rector/FunctionLike/ReturnTypeDeclarationRector/Fixture/skip_anonymous_function_nested_type.php.inc

- https://github.com/rectorphp/rector-src/blob/main/rules-tests/TypeDeclaration/Rector/FunctionLike/ReturnTypeDeclarationRector/Fixture/skip_iterable_array_iterator_co_type.php.inc

- https://github.com/rectorphp/rector-src/blob/main/rules-tests/TypeDeclaration/Rector/FunctionLike/ReturnTypeDeclarationRector/Fixture/skip_iterator_already.php.inc